### DISCUSSION TITLE
Inquiry about pure MM calculation (without PB, GB or 3D-RISM)

### DISCUSSION THREAD

**Discussion by srdjan125** [Valdes-Tresanco-MS/gmx_MMPBSA#310] [Q&A] [answered]
Hello,

I would like to perform only MM calculations without solvation contribution, in order to see pure enthalpy effects and also because I have coarse-grained MD trajectory, so I don't know how much does it make sense to do Poisson-Boltzmann solver on CG structures.

Would it work if I prepare same input file like for PB calculation, but just delete the part:

/
&pb
radiopt=0, istrng=0.150,
/

Or I would need some special command for it?   Thank you.

**Comment by Valdes-Tresanco-MS** [comment, marked as answer]
No, **gmx_MMPBSA** is designed to perform at least one of these calculations: PB, GB, 3D-RISM, or NMODES. To compute only **E<sub>int</sub>+E<sub>vdw</sub>+E<sub>ele</sub>** you can use Gromacs directly through `gmx energy`.

**Reply by srdjan125** [reply]
Hi, thank you for answer. Yes, I can use gmx energy for that, by creating energy groups according to index.ndx and doing some post processing. 
If gmx_MMPBSA would allow it it would be a bit less work for me :). 
But actually it seems worthy to try NMODEs and see can I get something interesting regarding entropy.

**Reply by Valdes-Tresanco-MS** [reply]
Note that gmx_MMPBSA requires all-atoms trajectories. Gromacs also includes normal modes calculations.

**Comment by srdjan125** [comment]
Ah thanx for information. Then I will use gmx_mmpbsa only for all-atom trajectories (CG trajectories backmappped to all-atom ones).

________________________________
